src/rule_tester.rs

```diff
@@ -123,44 +141,85 @@ fn assert_that_violations_match_expected(
         invalid_test.code
     );
     let mut violations = violations.to_owned();
-    violations.sort_by_key(|violation| violation.range);
+    // if https://github.com/tree-sitter/tree-sitter/pull/2460 gets merged then
+    // could restore this to `violations.sort_by_key(|violation| violation.range)`
```

Comment:
I can delete this comment, my PR got closed because (I think logically) their response was "actually it probably doesn't make sense for `Range` to implement `PartialOrd`/`Ord` at all"

(the context here is that I was seeing this sorting children before their parents (if both started on the same byte) which seemed unexpected)